README.md

```diff
@@ -23,32 +23,46 @@ Usage
 	apply plugin: 'marathon-deploy'
 
     marathon {
-	url = "http://path-to-your-marathon-instance.com"
+	    url = 'http://path-to-your-marathon-instance.com'
+	    dockerImageName = 'yourorg/app:1.0.0'
     }
+```
+
+Also [alenkacz/marathon-deployer](https://hub.docker.com/r/alenkacz/marathon-deployer/) Docker image is available so you can actually use this plugin to deploy any application from any environment where Docker is available:
+```bash
+docker run \
+    -e MARATHON_URL=http://path-to-your-marathon-instance.com \
+    -e DOCKER_IMAGE_NAME=yourorg/app:1.0.0 \
+    -v /path/to/your/marathon.json:/marathon.json
+    alenkacz/marathon-deployer
+```
 
 Properties
-====================
-- *url* - url to your marathon application
-- *dockerImageName* (OPTIONAL) - full image name that will be provided to marathon, use this only if you want to override the image name provided in your json (e.g. you want to deploy a testing version as a part of your CI/CD pipeline)
-- *pathToJsonFile* (OPTIONAL) - project relative path to your json file, default is *deploy/marathon.json*
-- *verificationTimeout* (OPTIONAL) - timeout when querying Marathon to verify that there are no pending deployments left, default is 90 seconds
-- *deploymentRequestTimeout* (OPTIONAL) - timeout for the initial deployment request, default is 5 seconds
-- *jvmOverhead* (OPTIONAL) - number of MBs that the container needs on top of JVM app memory (default is 200)
+==========
+| Gradle property | Docker environment variable | Default | Description |
```

Comment:
when previewing the table it does not look that good on my computer. The last column with the comments is too small :(